.github/workflows/WindowsTest.yml

```diff
@@ -35,12 +35,12 @@ jobs:
           name: ginger-artifact
 
 # ------------------------------------------------------------------------------------------------------
-# Install .Net core SDK 6.0.301
+# Install .Net core SDK 8.x.x
 # ------------------------------------------------------------------------------------------------------
       - name: Setup .NET
         uses: actions/setup-dotnet@v3
         with:
-          dotnet-version: 7.x.x
+          dotnet-version: 8.x.x
```

Comment:
> Consider specifying a more precise version of the .NET Core SDK instead of '8.x.x' to ensure consistent builds across all environments.

Please change it to 8.0.100 for all files